test/vfd_swmr_bigset_writer.c

```diff
@@ -370,11 +375,26 @@ state_init(state_t *s, int argc, char **argv)
     s->chunk_dims[0] = s->rows;
     s->chunk_dims[1] = s->cols;
     s->one_dee_max_dims[0] = s->rows;
-    s->one_dee_max_dims[1] = H5S_UNLIMITED;
+    if(s->fixed_array) {
```

Comment:
Write description/comments for the overall changes you make to vfd_swmr_bigset_writer.c for fixed array e.g. why the handling is different from current coding.

Reply:
I added some comments.  Hopefully, they are sensible.